app/js/lib/foxbox/recipes.js

```diff
@@ -28,8 +28,9 @@ export default class Recipe {
       `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/services`)
       // Get all recipes.
       .then(services => services.filter(service =>
-        service.adapter === 'thinkerbell-adapter' &&
-        service.id !== 'thinkerbell-root-service'
+        service.adapter != 'thinkerbell-adapter'
+        && service.adapter != 'webpush@link.mozilla.org'
```

Comment:
Honestly, I'd prefer this one, it's easier to read and see that all statements belong to the same condition, regarding git diff - if we add new conditions we just add more lines and previous lines should not be affected 
```
service.adapter != 'thinkerbell-adapter'
  && service.adapter != 'webpush@link.mozilla.org'
  && service.adapter != 'console@link.mozilla.org'
```